packages/gallery/src/components/item/videos/videoItemWrapper.js

```diff
@@ -63,8 +64,11 @@ class VideoItemWrapper extends ImageItem {
       'settings',
       'actions',
     ]);
+    const shouldCreatePlaceHoler =
+      utils.isSingleItemHorizontalDisplay(styleParams) && 
+      styleParams.videoPlay === GALLERY_CONSTS.videoPlay.AUTO;
 
-    return (
+    return shouldCreatePlaceHoler ? null : (
```

Comment:
How many components are rendering the video item wrapper? Can you pass the shouldCreatePlaceholder (state maybe) to an upper level? And this component will always do one thing(render) ?

Reply:
the video item wrapper is rendered on every video in the gallery, I thought it was the best place to put the shouldCreatePlaceholder since it is specific to videos (it should actually be called shouldCreateVideoPlaceholder)